.editorconfig

```diff
@@ -0,0 +1,12 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
```

Comment:
Tempted to leave this as just `*.java` - thoughts?

Reply:
I checked a few other projects, they seem to use `[*]`

Also, I recommend adding:
```
[*.md]
trim_trailing_whitespace = false
```

Because in Markdown there is `  ` (two trailing spaces) to break the line

Reply:
Plus `trim_trailing_whitespace = true` to `[*]`

Reply:
Good idea, will do. Thanks!

Reply:
Done

Reply:
I think it's good to have lf for all files, especially bash files, to make life easier for Windows development (currently we've solved this with the `.gitattributes`). And this might also help us in achieving remote caching for Windows builds for local development (not for CI though?).

Reply:
Hm no, it doesn't help with Windows cache, since it's only applied for edited files? But it's fine, we can solve it another way some day ;)